CHANGELOG

```diff
@@ -1,6 +1,6 @@
 # 3.10.4 (2024-XX-XX)
 
- * n/a
+ * Add sequence & mapping tests.
```

Comment:
```suggestion
 * Add `sequence` and `mapping` tests
```

Reply:
Done in https://github.com/twigphp/Twig/pull/3859/commits/9ecaf164fcf9de0a2f782153dbfa235d88a4efb6